resources/ext/cli_defaults/cli-defaults.sh.erb

```diff
@@ -4,4 +4,4 @@ if [ -n "$JRUBY_JAR" ]; then
   echo "Warning: the JRUBY_JAR setting is no longer needed and will be ignored." 1>&2
 fi
 
-CLASSPATH="${CLASSPATH}:/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/jars/*"
+CLASSPATH="${CLASSPATH}:/opt/puppetlabs/puppet/lib/ruby/vendor_ruby/facter.jar:/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/jars/*"
```

Comment:
@puppetlabs/release-engineering is there an ezbake variable that I should be interpolating here to get Facter's jar path?

Reply:
Not that I know of? I think this is all the ezbake variables: https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/staging-templates/ezbake.rb.mustache ?

Reply:
danke